ppoprf/src/ppoprf.rs

```diff
@@ -84,12 +85,33 @@ impl ProofDLEQ {
 pub type ServerPublicKey = Vec<RistrettoPoint>;
 
 // The wrapper for PPOPRF evaluations (similar to standard OPRFs)
-#[derive(Serialize, Deserialize)]
+#[derive(Serialize)]
 pub struct Evaluation {
+    #[serde(serialize_with = "ristretto_serialize")]
     output: CompressedRistretto,
     proof: Option<ProofDLEQ>,
```

Comment:
No critical since we're ignoring this part, but we probably want base64 serialization of the ProofDLEQ members too. That's a pair of `curve25519_dalek::scalar::Scalar` which are also 32-byte binary numbers, although differently-typed. The external serializer begins to look more attractive.